packages/grafana-ui/src/components/Tags/TagList.tsx

```diff
@@ -7,33 +7,45 @@ export interface Props {
   onClick?: OnTagClick;
   /** Custom styles for the wrapper component */
   className?: string;
+  'aria-label'?: string | ((name: string, i: number) => string);
 }
 
-export const TagList: FC<Props> = memo(({ tags, onClick, className }) => {
+export const TagList: FC<Props> = memo(({ tags, onClick, className, 'aria-label': ariaLabel }) => {
```

Comment:
When you move this to another PR 😉 , perhaps this prop should be renamed to like `getAriaLabel` to reduce confusion around "when can i pass a function or not?"?